### PR TITLE
[Docs] Japanese translation of docs/one_shot_keys.md

### PR DESCRIPTION
**Pull request by umi-umi**
## Description

Add a Japanese translation of docs/one_shot_keys.md.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@shelaf @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

File: docs/ja/one_shot_keys.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+```c
+#define ONESHOT_TAP_TOGGLE 5  /* この回数をタップすると、もう一度タップするまでキーが保持されます。*/
+#define ONESHOT_TIMEOUT 5000  /* ワンショットキーが解除されるまでの時間 (ms) */
+```
+
+* `OSM(mod)` - *mod*を一時的に押し続けます。[モッドタップ](ja/mod_tap.md)で示したように、`KC_*` コードでは無く、`MOD_*` キーコードを使わなければなりません。
+* `OSL(layer)` - 一時的に*レイヤー*に切り替えます。
+
+ワンショットキーをマクロあるいはタップダンスルーチンとして有効にしたい場合があります。
```
Comment (shelaf) [inline docs/ja/one_shot_keys.md:26]:
```suggestion
ワンショットキーをマクロあるいはタップダンスルーチンの一部として有効にしたい場合があります。
```

File: docs/ja/one_shot_keys.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+ワンショットレイヤーについては、キーダウン時に `set_oneshot_layer(LAYER, ONESHOT_START)` を呼び出し、キーアップ時に `clear_oneshot_layer_state(ONESHOT_OTHER_KEY_PRESSED)` を呼び出す必要があります。ワンショットをキャンセルする場合は、`reset_oneshot_layer()` を呼んでください。
```
Comment (shelaf) [inline docs/ja/one_shot_keys.md:28]:
```suggestion
ワンショットレイヤーについては、キーを押した時に `set_oneshot_layer(LAYER, ONESHOT_START)` を呼び出し、キーを放した時に `clear_oneshot_layer_state(ONESHOT_OTHER_KEY_PRESSED)` を呼び出す必要があります。ワンショットをキャンセルする場合は、`reset_oneshot_layer()` を呼び出してください。
```

File: docs/ja/one_shot_keys.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+  if (mods & MOD_MASK_GUI) {
+    println("Oneshot mods GUI");
+  }
+  if (!mods) {
+    println("Oneshot mods off");
+  }
+}
+```
+
+`mods` の引数は変更後のアクティブな mod が含まれるため、現在の状態が反映されます。
```
Comment (shelaf) [inline docs/ja/one_shot_keys.md:60]:
```suggestion
`mods` 引数は変更後のアクティブな mod が含まれるため、現在の状態が反映されます。
```

File: docs/ja/one_shot_keys.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+独自のキーボードを作成している場合、`_kb` と同等の機能もあります:
+
+```c
+void oneshot_locked_mods_changed_kb(uint8_t mods);
+void oneshot_mods_changed_kb(uint8_t mods);
+void oneshot_layer_changed_kb(uint8_t layer);
+```
+
+他のコールバックと同様に、更にカスタマイズを可能にするために `_user` の変種を呼ぶようにしてください。
```
Comment (shelaf) [inline docs/ja/one_shot_keys.md:107]: 変種はさすがに変なので、他のドキュメントでもあったバージョンとした方が良さそうです
```suggestion
他のコールバックと同様に、更にカスタマイズを可能にするために `_user` バージョンを呼ぶようにしてください。
```

File: docs/ja/one_shot_keys.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
+# ワンショットキー
+
+<!---
+  original document: 0.9.34:docs/one_shot_keys.md
+  git diff 0.9.34 HEAD -- docs/one_shot_keys.md | cat
+-->
+
+ワンショットキーは次のキーが押されるまでアクティブのままになり、その後放されるキーです。これにより一度に1つ以上のキーを押すことなく、キーボードの組み合わせを入力することができます。これらのキーは通常「スティッキーキー」あるいは「デッドキー」と呼ばれます。
```
Comment (mtei) [inline docs/ja/one_shot_keys.md:8]: 切れ目がわかりにくい気がするので、「後」を仮名に開いてみます。
```suggestion
ワンショットキーは次のキーが押されるまでアクティブのままになり、そのあと放されるキーです。これにより一度に1つ以上のキーを押すことなく、キーボードの組み合わせを入力することができます。これらのキーは通常「スティッキーキー」あるいは「デッドキー」と呼ばれます。
```

**Comment by mtei** [comment]
I think this is ready to merge.

**Comment by mtei** [comment]
Thanks!

**Pull request merged**